roles/fio_distributed/templates/server_vm.yml.j2

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  name: 'fio-server-{{item | string}}-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+  labels:
+    app: fio-benchmark-{{ trunc_uuid }}
+spec:
+  domain:
+    cpu:
+      cores: {{ workload_args.vm_cores | default(1) }}
+    devices:
+      disks:
+        - name: registrydisk
+        - name: cloudinitdisk
+{% if workload_args.storageclass is defined
+    or hostpath is defined %}
+        - name: data-volume
+          serial: data
+{% endif %}
+    resources:
+      requests:
+        memory: {{ workload_args.vm_memory | default(5G) }}
+  volumes:
+    - name: registrydisk
+      containerDisk:
+        image: {{ workload_args.vm_image | default('kubevirt/fedora-cloud-container-disk-demo:latest') }}
+    - name: cloudinitdisk
+      cloudInitNoCloud:
+        userData: |-
+          #cloud-config
+          password: fedora
+          chpasswd: { expire: False }
+          bootcmd:
+{% if workload_args.storageclass is defined
+    or hostpath is defined %}
+            - "mkdir -p {{ fio_path }} || true"
+            - mkfs.ext4 /dev/disk/by-id/ata-QEMU_HARDDISK_data
```

Comment:
Do we want to enforce ext4?

Reply:
Good comment... Actually we should make it possible to make a choice here for either a formatted drive with a choice of Filesystem OR using the drive as a bare block device...
Just didn't have time to implement this choice in this very first commit ;)

Reply:
@rsevilla87 can we continue with the PR as is, or do you need me to make the FS configurable?
IMO this is a secondary item that can be implemented by someone who actually needs this, but if you want this, I'll invest the time to get the PR merged

Reply:
Ok, makes sense, it could be implemented in the future.